packages/components/app/styles/components/alert.scss

```diff
@@ -113,7 +113,7 @@
   }
 
   > * + * {
-    margin-left: 8px;
+    margin-left: 16px;
```

Comment:
suggestion, non blocking: maybe convert this to a `gap` now that we can (finally, officially) use it? :)

Reply:
heh I did try it out but...well, try and see what it does :)

Reply:
Did you try adding to here?

```
.hds-alert__actions {
  gap: 16px;
}
```

Then also delete: 
```
// .hds-alert__actions > * + * {
//     margin-left: 16px;
// }
```